test/studies/lcals/RunVectorizeOnlyRawLoops.chpl

```diff
@@ -0,0 +1,501 @@
+            const len_minus_coeff = len - coefflen;
+            var val = 0;
+
+            ltimer.start();
+            for isamp in 0..#num_samples {
+              for i in 0..#len_minus_coeff {
+                var sum = 0.0;
+                for j in 0..#coefflen {
+                  sum += coeff[j]*input[i+j];
+                }
+                output[i] = sum;
+              }
+              val = isamp;
+            }
+            ltimer.stop();
+            loop_data.scalar_Real[0] = (val + 0.00123) / (val - 0.00123);
+            loopFinalize(iloop, stat, ilength);
+          }
+          when LoopKernelID.INIT3 {
+            loopInit(iloop, stat);
+            var out1 => loop_data.RealArray_1D[0];
+            var out2 => loop_data.RealArray_1D[1];
+            var out3 => loop_data.RealArray_1D[2];
+            var in1  => loop_data.RealArray_1D[3];
+            var in2  => loop_data.RealArray_1D[4];
+            ltimer.start();
+            for isamp in 0..#num_samples {
+              for i in vectorizeOnly(0..#len) {
+                const res = -in1[i] - in2[i];
+                out3[i] = res;
+                out2[i] = res;
+                out1[i] = res;
+              }
+            }
+            ltimer.stop();
+            loopFinalize(iloop, stat, ilength);
+          }
+          when LoopKernelID.MULADDSUB {
+            loopInit(iloop, stat);
+            var out1 => loop_data.RealArray_1D[0];
+            var out2 => loop_data.RealArray_1D[1];
+            var out3 => loop_data.RealArray_1D[2];
+            var in1  => loop_data.RealArray_1D[3];
+            var in2  => loop_data.RealArray_1D[4];
+            ltimer.start();
+            for isamp in 0..#num_samples {
+              for i in vectorizeOnly(0..#len) {
+                out1[i] = in1[i] * in2[i];
+                out2[i] = in1[i] + in2[i];
+                out3[i] = in1[i] - in2[i];
+              }
+            }
+            ltimer.stop();
+            loopFinalize(iloop, stat, ilength);
+          }
+          when LoopKernelID.IF_QUAD {
+            loopInit(iloop, stat);
+            var a  => loop_data.RealArray_1D[0];
+            var b  => loop_data.RealArray_1D[1];
+            var c  => loop_data.RealArray_1D[2];
+            var x1 => loop_data.RealArray_1D[3];
+            var x2 => loop_data.RealArray_1D[4];
+            ltimer.start();
+            for isamp in 0..#num_samples {
+              for i in vectorizeOnly(0..#len) {
+                var s = b[i]*b[i] - 4.0*a[i]*c[i];
+                if s >= 0 {
+                  s = sqrt(s);
+                  x2[i] = (-b[i]+s) / (2.0*a[i]);
+                  x1[i] = (-b[i]-s) / (2.0*a[i]);
+                } else {
+                  x2[i] = 0.0;
+                  x1[i] = 0.0;
+                }
+              }
+            }
+            ltimer.stop();
+            loopFinalize(iloop, stat, ilength);
+          }
+/*
+          when LoopKernelID.TRAP_INT {
```

Comment:
Why is this one commented out for the vectorizeOnly variant, but not the others?